### ISSUE TITLE
ANY, SOME, ALL support

### ISSUE THREAD

**Issue by mjburghoffer** [oguimbal/pgsql-ast-parser#70]
Currently it is impossible to write queries such as:

```sql
SELECT * FROM my_table WHERE my_column = ANY(ARRAY[1, 2, 3]);
```

Support for these types of operations would be very helpful!

see: https://www.postgresql.org/docs/13/functions-comparisons.html#id-1.5.8.30.16

**Comment by janpaepke** [comment]
So it turns out `any` is already supported.

The parsed result of your above query is as I would expect:
```javascript
{
  columns: [
    {
      expr: {
        name: '*',
        type: 'ref',
      },
    },
  ],
  from: [
    {
      name: {
        name: 'my_table',
      },
      type: 'table',
    },
  ],
  type: 'select',
  where: {
    left: {
      name: 'my_column',
      type: 'ref',
    },
    op: '=',
    right: {
      args: [
        {
          expressions: [
            {
              type: 'integer',
              value: 1,
            },
            {
              type: 'integer',
              value: 2,
            },
            {
              type: 'integer',
              value: 3,
            },
          ],
          type: 'array',
        },
      ],
      function: {
        name: 'any',
      },
      type: 'call',
    },
    type: 'binary',
  },
}
```

adding `some` (which is an alias for any) and `all` should be easy enough.

PR coming up.

**Comment by oguimbal** [comment]
Released as `pgsql-ast-parser@10.0.2`  thanks to @janpaepke  PR !
